doc/manpages/dinit-service.5.m4

```diff
@@ -168,7 +168,7 @@ That means this path is static and cannot itself have variable substitutions (se
 Indicates whether the service should automatically restart if it stops, including due to
 unexpected process termination or a dependency stopping.
 Note that if a service stops due to user request, automatic restart is inhibited.
-The default is to automatically restart.
+The default is to restart=$$$DEFAULT_AUTO_RESTART@@@.
```

Comment:
I changed its words but it was better before. I didn't find better words to integrate with that variable.

Reply:
Uhh, seems like `ifelse()` isn't processed in our .m4 files. man pages just print `ifelse(DEFAULT_AUTO_RESTART, true, The default is to automatically restart, The default is to not automatically restart)` but it works in my m4 interactive shell.

Reply:
If it works in the interactive shell, that should mean that it works generally...

Perhaps you're not (un)quoting it right? It needs `$$$` before the entire construct and `@@@` afterwards, something like

```
$$$ifelse(DEFAULT_AUTO_RESTART, true, @@@The default is to automatically restart$$$, @@@The default is to not automatically restart$$$)@@@
```

Reply:
I thought `$$$` is for resolving variables but it doesn't, interesting. I changed it to:
```m4
$$$ifelse(DEFAULT_AUTO_RESTART, true, The default is to automatically restart.,
    The default is to not automatically restart.)
@@@
```
Seems OK.